src/app/story/directives/basic.component.ts

```diff
@@ -29,9 +29,14 @@ import { TabService } from 'ngx-prx-styleguide';
       <hr/>
 
       <prx-fancy-field required label="Audio Files">
-        <prx-spinner *ngIf="!story?.versions"></prx-spinner>
-        <publish-upload *ngFor="let v of story?.versions" [version]="v" [strict]="strict"></publish-upload>
-        <h1 *ngIf="story?.versions?.length === 0">
+        <prx-select *ngIf="versionTemplateOptions" placeholder="Select Templates..."
+          [selected]="versionTemplatesSelected" [options]="versionTemplateOptions"
+          (onSelect)="updateVersions($event)"
+          [class.invalid]="versionsInvalid"
+          [class.changed]="versionsChanged"></prx-select>
+        <prx-spinner *ngIf="!undeletedVersions"></prx-spinner>
+        <publish-upload *ngFor="let v of undeletedVersions" [version]="v" [strict]="strict"></publish-upload>
+        <h1 *ngIf="undeletedVersions?.length === 0">
           You have no audio templates for this episode. How did that happen?
```

Comment:
<img width="724" alt="screen shot 2017-09-21 at 9 29 57 am" src="https://user-images.githubusercontent.com/2250413/30701380-bd725aca-9eaf-11e7-9aab-1ba2344195c3.png">
Should we keep this cheeky message? If yes, we should probably format it as a fancy field `hint`.

Reply:
Yeah - it does look a bit funny next to that other hint.  Will change that - and make it slightly less cheeky.  _Slightly_.